### ISSUE TITLE
stuck transfer detection may still not be working correctly on CCXTAdapter

### ISSUE THREAD

**Issue by cryptoeax** [cryptoeax/arbbot#116]
@andygoossens reported:

```
 "Notice: Undefined index: applyTime" @ https://github.com/cryptoeax/arbbot/blob/binance/bot/CCXTAdapter.php#L428
```

**Comment by andygoossens** [comment]
"applyTime" is only present in withdrawal history. Deposit history has a field "insertTime".

https://github.com/binance-exchange/binance-official-api-docs/blob/master/wapi-api.md#deposit-history-user_data

**Comment by cryptoeax** [comment]
Thanks!

**Comment by cryptoeax** [comment]
Fixed in d931ebe.
